patches/minecraft/net/minecraft/client/gui/GuiScreen.java.patch

```diff
@@ -35,15 +35,21 @@
          {
              GlStateManager.func_179101_C();
              RenderHelper.func_74518_a();
-@@ -418,6 +428,7 @@
+@@ -414,10 +424,13 @@
+ 
+     public void func_175281_b(String p_175281_1_, boolean p_175281_2_)
+     {
++        p_175281_1_=net.minecraftforge.event.ForgeEventFactory.onClientSendMessage(p_175281_1_);
++        if(p_175281_1_.isEmpty())return;
```

Comment:
@liach done. One moment while genPatches runs, then I will commit and push the change.